gaffer-as-a-service/gaas-rest/src/main/java/uk/gov/gchq/gaffer/gaas/factories/GafferFactory.java

```diff
@@ -76,10 +81,7 @@ private static GafferSpec overrideGafferSpecConfig(final GafferSpec config, fina
             config.putNestedObject(overrides.getSchema(), SCHEMA_FILE_KEY);
         }
 
-        // Mandatory Ingress values
```

Comment:
This undoes https://github.com/gchq/gaffer-experimental/pull/113/commits/784a4de95dff1b0feb6c4d414062d7d4550eec73